python_scripts/run_wandb.py

```diff
@@ -0,0 +1,37 @@
+import os, sys, subprocess
+import wandb
+
+all_args = sys.argv[1:]
+
+config_files_dict = {
+    'BarlowTwins': 'cc_BarlowTwins.yaml',
+    'byol': 'cc_byol.yaml',
+    'SimCLR': 'cc_SimCLR.yaml',
+    'spectralReg': 'cc_spectralReg.yaml',
+}
+seeds = [1, 2, 3]
+
+wandb.init(entity='eigengroup', project='fastssl')
+
+ssl_train_cmd = 'python python_scripts/train_model.py'
+for arg in all_args:
+    ssl_train_cmd += ' {}'.format(arg)
+    if 'algorithm' in arg:
+        assert arg.split('=')[1] in config_files_dict.keys(), "SSL Algorithm {} not implemented".format(arg.split('=')[1]) 
+        ssl_train_cmd += ' --config-file configs/cc_{}.yaml'.format(arg.split('=')[1])
+print(ssl_train_cmd)
+os.system(ssl_train_cmd)
+
+for seed in seeds:
```

Comment:
This probably needs to be changed now because we want to run 3 seeds of SSL pretraining and corresponding linear eval with the same seed. Hence, put the `ssl_train_cmd` part (L16-L23) inside this loop.
Ok for now, but make change in future PR.